source/manual/retiring-an-application.html.md

```diff
@@ -134,10 +134,14 @@ Some applications are responsible for publishing certain routes. If you're
 retiring a publishing application, make sure you check if any of its content
 items need to be unpublished and do it via the Publishing API.
 
-## 17. Move repo
+## 17. Remove from Errbit / Sentry
+
+Since the application has been retired, it shouldn't be tracked in Errbit or Sentry. 
```

Comment:
Are there any docs about how we do this?  I was able to do sentry, but I think only because I have some elevated permissions, no idea how to do it on errbit.

Reply:
I couldn't find any documentation on how to remove an app from Sentry, except for the API version. 

I've removed the errbit part since it's being retired.

Reply:
Cool.

The commit message still references errbit though.  Change that and you're good to merge I reckon.